repair_refurbish/models/repair.py

```diff
@@ -8,9 +8,7 @@ class RepairOrder(models.Model):
     _inherit = "repair.order"
 
     to_refurbish = fields.Boolean()
-    location_dest_id = fields.Many2one(
-        string="Delivery Location", comodel_name="stock.location"
-    )
+    location_dest_id = fields.Many2one(string="Delivery Location", readonly=False)
```

Comment:
comodel_name removed? why?

Reply:
Field is already there in Odoo repair module, that's why

Reply:
I see, thanks for clarifying